src/app/workflow/versions/versions.component.html

```diff
@@ -118,6 +118,24 @@
       </td>
     </ng-container>
 
+    <ng-container matColumnDef="engineVersions">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        matTooltip="The engine version used by the version's descriptor file(s)."
```

Comment:
Nit, I suggest "The minimum required Nextflow version." or "The minimum required Nextflow engine version". See https://www.nextflow.io/docs/latest/config.html#scope-manifest